.ci/ci_entry_point.sh

```diff
@@ -37,6 +37,8 @@ repo_to_test="${1:-}"
 repo_to_test=${repo_to_test#"https://"}
 repo_to_test=${repo_to_test%".git"}
 
+# test type pass to jenkins_job_build.sh
+TEST_TYPE="${2:-all}"
```

Comment:
nit: maybe update the "usage" comment at the top, to include this optional parameter?

Reply:
Good point! thanks.